apps/backend/src/modules/chat-messages/chat-message.repository.ts

```diff
@@ -188,6 +188,62 @@ class ChatMessageRepository implements Repository<ChatMessageEntity> {
 		});
 	}
 
+	public async setReadChatMessages(
+		chatMessageIds: number[],
+	): Promise<ChatMessageEntity[]> {
+		const updatedChatMessages = await this.chatMessageModel
+			.query()
+			.whereIn("id", chatMessageIds)
+			.update({ status: MessageStatus.READ })
+			.withGraphFetched(
+				`${RelationName.SENDER_USER}.[${RelationName.USER_DETAILS}.${RelationName.AVATAR_FILE}, ${RelationName.GROUPS}.${RelationName.PERMISSIONS}]`,
+			)
+			.returning("*")
```

Comment:
```suggestion
			.updateAndFetch({ status: MessageStatus.READ })
			.withGraphFetched(
				`${RelationName.SENDER_USER}.[${RelationName.USER_DETAILS}.${RelationName.AVATAR_FILE}, ${RelationName.GROUPS}.${RelationName.PERMISSIONS}]`,
			)
```

Try updateAndFetch, so you will not need to use returning("*")